src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/types';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
```

Comment:
Consider naming your type `TodoListProps` for better readability, and more explicit naming. 📚